chapter_04/svv/exercise1.pl

```diff
@@ -0,0 +1,20 @@
+use strict;
+use warnings;
+
+require "total.pl";
```

Comment:
Using ``require`` to import code is an interesting approach. There are some considerations with this which we'll see later, so watch out if you try to use this in production.